scripts/build/bundler.js

```diff
@@ -176,13 +176,14 @@ function getWebpackConfig(bundle) {
       path: path.resolve(root, "dist"),
       filename: bundle.output,
       library: ["prettierPlugins", bundle.name],
-      libraryTarget: "umd"
+      libraryTarget: "umd",
+      // https://github.com/webpack/webpack/issues/6642
+      globalObject: 'new Function("return this")()'
     },
-    plugins: [
-      new webpack.DefinePlugin({
-        "process.env.NODE_ENV": JSON.stringify("production")
-      })
-    ]
+    optimization: {
+      // disable terser for `parser-postcss.js`
+      minimize: false
```

Comment:
Why we disable this?

Reply:
this module can't be minified, unknown reason, we can figure later

https://github.com/prettier/prettier/pull/6200#issuecomment-500018106

Reply:
may be I should add a comment to this link or a TODO

Reply:
@fisker yes, please leave comment in source code `// Todo description`